c7n/resources/dynamodb.py

```diff
@@ -450,3 +454,235 @@ class resource_type(object):
         name = 'TableName'
         date = 'CreationDateTime'
         dimension = 'TableName'
+
+
+@resources.register('dax')
+class DynamoDbAccelerator(query.QueryResourceManager):
+
+    class resource_type(object):
+        service = 'dax'
+        type = 'cluster'
+        enum_spec = ('describe_clusters', 'Clusters', None)
+        detail_spec = None
+        id = 'ClusterArn'
+        name = 'ClusterName'
+        config_type = 'AWS::DAX::Cluster'
+        filter_name = None
+        dimension = None
+        date = None
+
+    retry = staticmethod(get_retry(('Throttled',)))
+    filters = FilterRegistry('dynamodb-dax.filters')
```

Comment:
filter_registry = FilterRegistry .. and you can omit the filter_registry = filters later